libfwupd/fwupd-security-attr.h

```diff
@@ -47,6 +50,9 @@ typedef enum {
 	FWUPD_SECURITY_ATTR_FLAG_RUNTIME_UPDATES = 1 << 8,
 	FWUPD_SECURITY_ATTR_FLAG_RUNTIME_ATTESTATION = 1 << 9,
 	FWUPD_SECURITY_ATTR_FLAG_RUNTIME_ISSUE = 1 << 10,
+	FWUPD_SECURITY_ATTR_FLAG_ACTION_OEM_FW_UPDATE = 1 << 11,
```

Comment:
Can you make this instead 
`FWUPD_SECURITY_ATTR_FLAG_ACTION_CONTACT_MANUFACTURER`

I think it would make it more clear.

Reply:
FWUPD_SECURITY_ATTR_FLAG_ACTION_CONTACT_OEM makes it clear to contact the OEM rather than the vendor who's name is on the chip, e.g. Broadcom/Intel. Fixed.